### ISSUE TITLE
"Tick less than 10s away!" bug in turn based games

### ISSUE THREAD

**Issue by Skilly146** [Tsangares/stoned_ape_tools#25]
## What

In turn based games the line 'Tick less than 10s away!' is always on screen since time to tick is always low

## How

I think I found the function that is adding this text at line 923 in intel.js
```javascript
if (universe.timeToTick(1).length < 3) {
        let lineHeight = 16 * map.pixelRatio;
        map.context.font = (14 * map.pixelRatio) + "px OpenSansRegular, sans-serif";
	map.context.fillStyle = "#FF0000";
        map.context.textAlign = "left";
        map.context.textBaseline = "middle";
        let s = "Tick < 10s away!";
        if (universe.timeToTick(1) === '0s') {
	        s = "Tick passed. Click production countdown to refresh.";
        }
        drawOverlayString(map.context, s, 1000, lineHeight);
```

It should be as simple as adding a statement saying `if turn_based then cancel`, either in the function or where it is called

**Comment by Tsangares** [comment]
Oh wow. Thank you

**Comment by Tsangares** [comment]
Easy fix:
```
if (!NeptunesPride.gameConfig.turnBased && universe.timeToTick(1).length < 3) {
				let lineHeight = 16 * map.pixelRatio;
				map.context.font = (14 * map.pixelRatio) + "px OpenSansRegular, sans-serif";
				map.context.fillStyle = "#FF0000";
				map.context.textAlign = "left";
				map.context.textBaseline = "middle";
				let s = "Tick < 10s away!";
				if (universe.timeToTick(1) === '0s') {
					s = "Tick passed. Click production countdown to refresh.";
				}
				drawOverlayString(map.context, s, 1000, lineHeight);
			}
			```
			Published in 2.21. Thank you Skilly
